setup/user/resources.go

```diff
@@ -0,0 +1,175 @@
+package user
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"sync"
+
+	cfg "github.com/codeready-toolchain/toolchain-e2e/setup/configuration"
+
+	"github.com/hashicorp/go-multierror"
+	errs "github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	"k8s.io/apimachinery/pkg/util/wait"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/discovery/cached/memory"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+)
+
+type templateProcessor struct {
```

Comment:
what about reusing the template processor in https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/template/processor.go instead?